helpers/voting/projects/polymarket.ts

```diff
@@ -226,3 +227,28 @@ export function maybeMakePolymarketOptions(
     ];
   }
 }
+
+/**
+ * Strips the following keys from ancillary data (and any possible preceding commas):
+ * 1. ooRequester
+ * 2. childRequester
+ * 3. childChainId
+ */
+
+// questionId is a hash of ancillaryData + initializer EOA
+export function sanitizeAncillaryData(decodedAncillaryData: string): string {
+  const regex = /(?:,\s*)?(ooRequester|childChainId|childRequester):[^,]+/g;
+  const stripped = decodedAncillaryData.replace(regex, "");
+  return stripped;
+}
+
+export function getQuestionId({
```

Comment:
seems like a simple solution, wonder why we didnt do this before

Reply:
Couldn't query PM by questionId before, this is a new feature they implemented for us.